crates/indexer/src/http/metadata_json.rs

```diff
@@ -419,6 +429,8 @@ pub async fn process<'a>(client: &Client, meta_key: Pubkey, uri_str: String) ->
         .chain(id.arweave.map(|a| a.0.to_vec()))
         .collect();
     let addr = bs58::encode(meta_key).into_string();
+    let first_verified_creator =
+        first_verified_creator.map(|address| bs58::encode(address).into_string());
```

Comment:
This logic should occur in `process_attributes`

Reply:
Don't have access to the creators in process_attributes. Need to grab at the time indexing the metadata like the address is being passed in currently.

Reply:
Can't we just pass the un-stringified version through to process_attributes and stringify it somewhere else?  This particular function it's in mainly serves as parsing and dispatch logic for the other process functions.

Reply:
Ended up eating my words, the fix for the other comment involved moving this logic back into this function.